packages/mds-policy-service/service/provider.ts

```diff
@@ -99,5 +100,16 @@ export const PolicyServiceProvider: ServiceProvider<PolicyService, PolicyService
       }
 
       return publishedPolicy
+    }),
+  writePolicyIntentToPolicy: (context, intent_draft) =>
+    serviceErrorWrapper('writePolicyIntentToPolicy', async () => {
+      const policy = translateIntentToPolicy(intent_draft)
+      const { policy_id } = policy
+      await PolicyRepository.writePolicy(policy)
+      await PolicyRepository.writePolicyMetadata({
+        policy_id,
+        policy_metadata: { intent_type: intent_draft.intent_type }
+      })
+      return PolicyRepository.publishPolicy(policy.policy_id)
```

Comment:
Why not use the destructured `policy_id` here?
```suggestion
      return PolicyRepository.publishPolicy(policy_id)
```